apps/projects/app/components/Content/IssueDetail/Markdown.js

```diff
@@ -11,7 +11,11 @@ const Link = ({ children, ...props }) => (
   </SafeLink>
 )
 
-const ListItem = ({ checked, children, ...props }) => {
+Link.propTypes = {
+  children: PropTypes.element.isRequired,
```

Comment:
I wasn't sure about this, either. The recommended linter settings required all propTypes to be defined, so I decided to just roll with it, rather than override. Since you asked about it here, and discovered that [this is intentional](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prop-types.md#rule-options): the official recommendation there is to add `children` to `propTypes`. It encourages using `PropTypes.node`, however, instead of `PropTypes.element`. I'll update this PR accordingly.